common/media.js

```diff
@@ -11,7 +11,10 @@ function getVideoURI(base)
 
     if ( videotag.canPlayType )
     {
-      if (videotag.canPlayType('video/webm; codecs="vp9, opus"') )
+      if ( videotag.canPlayType('video/mp4; codecs="avc1, mp4a.40.2"') )
```

Comment:
This isn't the appropriate codecs parameter for all the MP4 videos we have.

e.g.:

* `media/counting.mp4` is `video/mp4; codecs="mp4v.20.f3"`
* `media/2x2-green.mp4` is `video/mp4; codecs="avc1.64000A, mp4a.6B"`